scripts/language-file-to-csv/language-file-to-csv.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const fs = require('fs');
+
+if (process.argv.length < 3) {
+  console.error('ERROR - Language file path argument missing!');
+  return;
+}
+
+let languageFilePath = process.argv[2];
+let validLanguageFilePath = /site\/[a-z]{2}\_[A-Z]{2}.all.json/;
+
+if (!languageFilePath.match(validLanguageFilePath)) {
+  console.error(`ERROR - Language file path argument '${languageFilePath}' invalid!`);
+  return;
+}
+
+let rawdata = fs.readFileSync(languageFilePath);
+let json = JSON.parse(rawdata);
+
+// Remove keys where value is same as key.
+// These keys are (hopefully) not used by festival clients.
+for (const [key, value] of Object.entries(json)) {
+  if (key === value.other) {
+    console.warn(`WARNING - Unused key '${key}' will be removed!`);
+    delete json[key];
+  }
+}
+
+// Because this character '~' is not present in the English language file.
+// May have to change how this works in the future.
+let csvSeparator = '~'; 
```

Comment:
This seems a little dangerous. Perhaps there is some easy js you could use that makes sure strings are quoted and escaped properly. At the very least:
```js
let field = `"${value.replace(/"/g, '""')}"`
```